packages/video-intelligence/package.json

```diff
@@ -1,16 +1,15 @@
 {
-  "name": "@google-cloud/video-intelligence",
-  "version": "0.1.1",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "name": "@google-cloud/videointelligence",
+  "version": "0.1.0",
```

Comment:
I don't think the generator should ever modify our version. That's bumped during our release process, and is never part of a PR.